k8s/namespaces/wa/kustomization.yaml

```diff
@@ -7,4 +7,5 @@ bases:
 - wa-task-management-api/wa-task-management-api.yaml
 - wa-task-configuration-api/wa-task-configuration-api.yaml
 - wa-proto-frontend/wa-proto-frontend.yaml
+- wa-case-event-handler/wa-case-event-handler.yaml
```

Comment:
new apps should go to other envs first, adding here makes it go to all envs

Reply:
Hi @timja thanks for letting me know I've updated the PR to add it only to the 
`k8s/aat/common-overlay/wa/kustomization.yaml` file.
Is that the correct approach? Thank you